.github/workflows/infrastructure-repository-update.yml

```
@@ -695,9 +695,21 @@ jobs:
             exit 1
           fi
 
+  Redirect:
```

actionlint (1.7.9)

job "redirect" needs job "gradle" which does not exist in this workflow

(job-needs)